packages/presentation/lib/l10n/intl_ru.arb

```diff
@@ -0,0 +1,3 @@
+{
+  "share": "Смотри, какой я фильм нашел! \n\nhttps://www.themoviedb.org/movie/{movie_id}?language={language}"
+}
```

Comment:
Did u tested ru locale? My app crushed when I switched on to ru, cuz require all strings to be ru.

Reply:
yes, everything is adequate for me, only the share changes